backend/index.js

```diff
@@ -0,0 +1,71 @@
+import Config from "./config";
```

Comment:
This is the entry point of our API. All needed services are instantiated here and passed as arguments to other services ( dependency injection ).